queueConversationControl/queueConversationControl.html

```diff
@@ -0,0 +1,104 @@
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://use.fontawesome.com/2626cafbb6.js"></script>
+    <script src="node_modules/jquery/dist/jquery.js"></script>
+    <script src="node_modules/ms-signalr/jquery.signalR.js"></script>
+    <script src="../common/micc.js"></script>
+    <script src="../common/miccSignalR.js"></script>
+    <script src="queueConversationControl.js"></script>
+</head>
+
+<body>
+    <div class="w3-container w3-blue">
+        <h1 class="w3-text-shadow">MiCC Queue Conversation Control demo</h1>
+    </div>
+    <div class="w3-accordion w3-card-4">
+        <div class="w3-row">
+            <div class="w3-col m6">
+                <button class="w3-btn-block w3-blue w3-topbar w3-border-white" data-toggle="collapse" data-target="#login">Show login</button>
+                <div id="login" class="collapse">
+                    <p>
+                        <input class="w3-input w3-border w3-round" type="text" id="miccServer" placeholder="Server address"></input>
+                    </p>
+                    <p>
+                        <input class="w3-input w3-border w3-round" type="text" id="username" placeholder="User name"></input>
+                    </p>
+                    <p>
+                        <input class="w3-input w3-border w3-round" type="password" id="password" placeholder="Password"></input>
+                    </p>
+                    <p>
+                        <button class="w3-btn-block w3-blue w3-round" onclick="login();">Login</button>
+                    </p>
+                </div>
+            </div>
+            <div class="w3-col m6">
+                <button class="w3-btn-block w3-blue w3-topbar w3-border-white" data-toggle="collapse" data-target="#misc">Show Misc</button>
+                <div id="misc" class="collapse">
+                    <p>
+                        <div class="w3-row-padding">
+                            <div class="w3-col m6">
+                                <input class="w3-input w3-border w3-round" type="text" id="queueId" placeholder="ID, reporting, or name"></input>
+                            </div>
+                            <div class="w3-col m6">
+                                <button class="w3-btn-block w3-blue w3-round" onclick="addQueueRow($('#queueId').val());">Add queue</button>
+                            </div>
+                        </div>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="w3-panel w3-card-4">
+        <h4>Queue Stats</h4>
+        <table class="w3-table-all w3-centered" id="queueStatsTable">
+            <thead>
+                <tr class="w3-blue">
```

Comment:
Remove can be done without the stats.  The user will know that pick succeeded as the conv should disappear from the queue as it heads to an inbox.  We can discuss further when you demo the GUI to me.